TMDbLib/Rest/RestResponse.cs

```diff
@@ -53,7 +55,14 @@ public RestResponse(HttpResponseMessage response, RestClient client)
         {
             try
             {
-                return response.GetDataObject().Result;
+                if (response.Valid)
+                {
+                    return response.GetDataObject().Result;
+                }
+                else
+                {
+                    return default(T);
```

Comment:
In this case, I'd rather fail. IIRC, the RestResponse is never exposed -- so we are in complete control of calls. 

For this case, it should be "if (!Valid) throw new ..".

Reply:
See my comment in the PR thread about this line.